src/index.html

```diff
@@ -1,14 +1,65 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a class="header__logo" href="#home"><img alt="Logo" src="images/logo.png"></a>
+
+      <nav class="header__navigation nav">
+
+        <ul class="nav__table">
+
+          <li class="nav__item">
+            <a class="is-active nav__link APPLE" href="#">apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              data-qa="hover"
+              href="#"
+              class="nav__link"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link VIDEO">video</a>
```

Comment:
seems that you don't need class VIDEO here

Reply:
Done.